database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -21,6 +21,7 @@ public function up()
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
             $table->integer('permission')->default(0);
+            $table->boolean('verified')->default(false);
```

Comment:
Already merged migrations should not be updated. Every change needs a new migration because deployed versions run only the new ones, the database can not be rollbacked.

Reply:
You are right, moved to a new migration.